examples/java/build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
     dependencies {
-        classpath "org.jetbrains.kotlinx:kotlinx-benchmark-plugin:0.4.8"
+        classpath "org.jetbrains.kotlinx:kotlinx-benchmark-plugin:0.4.255-SNAPSHOT"
```

Comment:
Do I get right that it works on CI due to the tests are never being run with an overridden version parameter?

Reply:
I guess it's because the plugin is an included build here, so the specified version seems to be ignored.